src/hooks/useSyncBrandingSetting.ts

```diff
@@ -0,0 +1,12 @@
+import { useUpdateAtom } from 'jotai/utils'
```

Comment:
Instead of splitting the state and the hooks, if this is a one-off, how do you feel about simplifying it by just having a hook file that does not export an atom, eg

    const disableBrandingAtom = atom<boolean>(false)

    export function useDisableBranding(): [boolean, (value: boolean) => void] {
      return useAtom(disableBrandingAtom)
    }

Reply:
I like this idea a lot :)